src/test/resources/keep-comments-in-pom/expected-flattened-pom.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.codehaus.mojo.flatten.its</groupId>
+  <artifactId>resolve-properties-ci-do-not-interpolate-profile-activation-file</artifactId>
+  <version>1.2.3.4</version>
+  <properties>
+    <!-- some nice comment -->
+    <revision>1.2.3.4</revision>
+    <propertyWithoutComment>test-propertyWithoutComment</propertyWithoutComment>
+    <!-- comment1 for propertyWithTwoComments -->
+    <!-- comment2 for propertyWithTwoComments -->
+    <propertyWithTwoComments>test-propertyWithTwoComments</propertyWithTwoComments>
+  </properties>
+  <build>
+    <defaultGoal>verify</defaultGoal>
+    <plugins>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>flatten-maven-plugin</artifactId>
+        <configuration>
+          <flattenMode>resolveCiFriendliesOnly</flattenMode>
+        </configuration>
+      </plugin>
+      <plugin>
+        <!-- plugin with multiple same node names -->
+        <artifactId>maven-surefire-plugin</artifactId>
+        <configuration>
+          <excludes>
+            <exclude>**/TestCircle.java</exclude>
+            <exclude>**/TestCircle2.java</exclude>
+            <!-- special info for TestSquare -->
+            <exclude>**/TestSquare.java</exclude>
+            <!-- special info for TestSquare2 -->
+            <exclude>**/TestSquare2.java</exclude>
+          </excludes>
+        </configuration>
+      </plugin>
+    </plugins>
+  </build>
+  <profiles>
+    <profile>
+      <!-- another comment what the profile is doing -->
+      <activation>
+        <file>
+          <exists>file.txt</exists>
+        </file>
+      </activation>
+    </profile>
+    <profile>
+      <id>multiline-profile</id>
+      <!-- comment with
+				 multiple lines 
+				 in a profile -->
```

Comment:
The indentation here is incorrect.

Reply:
Indentation is not correct for testing it is kept like it is in the target pom.xml:
[expected-flattened-pom-jdk11.xml](https://github.com/mojohaus/flatten-maven-plugin/pull/270/files/863a925c5ab2156c4fb8ec02cb6b277fee5c8ea9#diff-e6ffaac4fd273ed9c3c17d5905a24d23306b48d18fcb63de999f3ec3e8d3f24c)